lib/msal-node/src/request/ManagedIdentityRequestParams.ts

```diff
@@ -5,10 +5,12 @@
 
 /**
  * ManagedIdentityRequest
+ * - claims       - a stringified claims request which will be used to determine whether or not the cache should be skipped
```

Comment:
```suggestion
 * - claims       - in case a resource refuses to authenticate with a token (e.g. because the token has been revoked by the tenant admin), it may send a 401 response and a WWW-Authenticate header with a claims parameter. When this is configured, MSAL always requests a new token from the identity provider, and will add the claims to the request where this is supported.
```